lbrynet/core/client/ClientProtocol.py

```diff
@@ -208,10 +208,11 @@ def get_next_request(results):
                         log.info(result.value)
                         log.info("The connection to %s is closing due to an error: %s",
                                  self.peer, result.getTraceback())
+
+                        self.peer.report_down()
```

Comment:
without this, connection manager would attempt to reconnect to a peer that was returning an improper response